experimental/src/main/java/org/jboss/bacon/experimental/impl/dependencies/DependencyResolver.java

```diff
@@ -55,6 +57,13 @@ public DependencyResolver(DependencyResolutionConfig dependencyResolutionConfig)
 
     private void setupConfig(ProjectDependencyConfig.Mutable dominoConfig) {
         config.getExcludeArtifacts().stream().map(GACTVParser::parse).forEach(dominoConfig::addExcludePattern);
+        DependencyExcluder dependencyExcluder = new DependencyExcluder(Config.instance().getActiveProfile().getDa());
```

Comment:
I think that creating new config class for Autobuilder would be better, then reusing a Dependency Analyzer config.

Also I wonder, if this config may be optional, so people not using autobuilder don't need it, but require it for the autobuilder.

Reply:
I'll create the autobuild class. I've written a specific test to make sure builds don't break if the configuration is missing.

Reply:
Done. I've created a new config class for autobuild. It is optional and shouldn't fail if isn't informed. I've created a test case for this situation and it seems to work correctly.